agent/src/main/scala/za/co/absa/atum/agent/model/MeasuresMapper.scala

```diff
@@ -17,24 +17,26 @@
 package za.co.absa.atum.agent.model
 
 import za.co.absa.atum.agent.model.Measure._
+import za.co.absa.atum.model.dto
 
 case class UnsupportedMeasureException(msg: String) extends Exception(msg)
```

Comment:
actually, consider (but I'll leave it up to you, just an idea) to move this to something like `za.co.absa.atum.agent.exception` module

Reply:
Moved